src/components/TableBody.js

```diff
@@ -11,6 +14,21 @@ class TableBody extends Component {
     var lockstate = this.props.lockstate || 'unlocked';
     var total_working_hours = this.props.totaldays * 8;
     var total = 0;
+    // clear weekends from data
+    var weekday = [];
+    var i = 0;
+    Object.values(data).forEach(value => {
+      console.log('index is : ' + i)
+      if (moment(value.event_date).isoWeekday() < 6) {
+      } else {
+        weekday.push(i)
+      }
+      i++;
+    });
+    console.log('weekday array is ' + weekday)
```

Comment:
This one too

Reply:
fixed in https://github.com/codelabsab/timereport-ui/pull/8/commits/968d0e637092d5286ac590562b5526da9fe0d1c9